src/steps/os/linux.rs

```diff
@@ -550,7 +557,15 @@ pub fn run_pacdef(ctx: &ExecutionContext) -> Result<()> {
             .args(["package", "review"])
             .status_checked()?;
     } else {
-        ctx.run_type().execute(&pacdef).arg("sync").status_checked()?;
+        ctx.run_type()
+            .execute(&pacdef)
+            .arg("sync")
+            .arg(if ctx.config().yes(Step::System) {
+                "--noconfirm"
```

Comment:
Seems like the new version (1.x) of `pacdef` also [supports the `--noconfirm` option](https://github.com/steven-omaha/pacdef/blob/73c041922fcf3d51f915008923842bb1ce9ac6a7/README.md?plain=1#L154):

```shell
$ pacdef package sync --noconfirm
```

Would you like to add it?

Reply:
Sorry for missing that, fixed